app/components/login/login.jade

```diff
@@ -0,0 +1,17 @@
+div.row
+  diw.col-md-6.col-md-offset-3
```

Comment:
`diw` - is a wrong tag. 
check http://jade-lang.com/reference/attributes/ Class Literal , if you use class, div is a tag by default and you can omit it in code